packager/media/base/stream_info.h

```diff
@@ -161,6 +166,10 @@ class StreamInfo {
   // repackaging an init segment alone.
   uint32_t default_sample_duration_ = 0;
 
+  // Flag to indicate whether the duration originated in the movie extends
+  // header and should carry over to the output segment.
+  bool mehd_header_carryover_ = false;
```

Comment:
The `mehd` box is specific to mp4 so I am not sure that it should belong here.  Additionally, why not change this to store the actual default fragment duration (e.g., `default_fragment_duration`, which is more generic) and add setter/getter for it to set/get the fragment duration subsequently in the `mvex` box directly?